src/iotjs_binding.c

```diff
@@ -85,6 +86,43 @@ double iotjs_jval_as_number(jerry_value_t jval) {
 }
 
 
+bool iotjs_jbuffer_get_bufferwrap(jerry_value_t jval,
```

Comment:
I still don't get this name: `iotjs_jbuffer_get_bufferwrap`. It is a function which converts a jerry string or a buffer to iotjs string. It does not return with a bufferwrap as the name suggests.

Other than that the patch is ok.